index.bs

```diff
@@ -779,6 +782,21 @@ spec:dom-ls; type:interface; text:Document
     dangerous and should not be used if you have Real Deal auth and csrf cookies
     used.
 
+   * '<dfn>`unsafe-include-credentials`</dfn>' All cross-origin requests to the
```

Comment:
I think might be worth clarifying that this includes not just XHR but image tags etc etc too. so 
<img crossorigin=anonymous> will be overridden to be img crossorigin=with-credentials